packages/utils/src/RenderIndicator.tsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-export const RenderIndicator: React.FC<any> = ({ style, parentDom }) => {
+export const RenderIndicator: React.FC<any> = ({
+  style,
+  parentDom,
+  ...rest
```

Comment:
Can be more specific than a spread, no major thoughts here.

Reply:
Do we really need `...rest` here? I don't think there's a use case for spreading the props yet.

Reply:
Removed, it was an artifact from a previous idea 👍